README.md

```diff
@@ -74,6 +74,16 @@ This following command generates an [Apache RAT](https://creadur.apache.org/rat/
 $ sbt ratCheck
 ```
 
+### sbt-scoverage
+
+This follow commands generate an [sbt-scoverage](https://github.com/scoverage/sbt-scoverage) test coverage report that will be located in ``target/scala-ver/scoverage-report/``.
+
+```bash
+$ sbt clean coverage test it:test
+$ sbt coverageReport
+$ sbt coverageAggregate
+```
+
```

Comment:
This will be really nice! Super easy to generate reports.

Can you also modify the .travis.yml file to add the coverage commands and upload the reports to codecov.io? According to this example, it only takes a couple extra lines:

https://github.com/codecov/example-scala/blob/master/.travis.yml

Reply:
That was the plan, I was just going to do it under a separate pull request, because I wasn't sure how straightforward it would be.

Reply:
That works for me.

Reply:
I ended up just adding it to the pull request once I saw it was just a couple lines